src/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import categoriesReducer from './redux/categories/categories';
+import bookReducer from './redux/books/books';
+
+const reducers = {
+  books: bookReducer,
+  categories: categoriesReducer,
+};
+
+const store = createStore(combineReducers(reducers));
+
+export default store;
```

Comment:
- [ ] please make sure that your `ConfigureStore.js` inside the `redux folder` in order to follow the ducks pattern 
![image](https://user-images.githubusercontent.com/61790587/173447499-fbf9123a-7f87-4462-9db1-19ee4ec96cef.png)